install_config/configuring_sdn.adoc

```diff
@@ -18,11 +18,31 @@ network_. Two xref:../architecture/additional_concepts/sdn.adoc#architecture-add
 are currently available (*ovs-subnet* and *ovs-multitenant*), which provide
 different methods for configuring the pod network.
 
+== Configuring the Pod Network with Ansible
```

Comment:
We seem to be lacking examples or guidance on configuring directives like the following in this section: 

**openshift_master_default_subdomain**, **osm_cluster_network_cidr**, **osm_host_subnet_length**, and **openshift_node_proxy_mode** 

all apply or can apply to the clusters networking configuration.